translations/ja.json

```
@@ -2211,6 +2213,38 @@
+      "countChipLabel": "{label} <strong>{count}</strong>",
+      "categoryDescriptions": {
+        "sensitiveCapabilities": "Permissions, entitlements, and service bindings that grant the app access to something sensitive — messages, contacts, location, microphone, or admin-level device control.",
+        "deviceManipulation": "Signals where the app can actively alter device state, persist itself, or be manipulated by other apps on the device — not just observe it.",
+        "monitoringIndicators": "Capabilities built for continuous, silent observation of the user rather than one-off access — usage tracking, notifications, screen capture.",
+        "controlBypass": "Signals that the app can sidestep the org's own store-review, network, or device-management controls.",
+        "suspiciousIndicators": "Heuristic and hygiene flags that read as suspicious in combination rather than individually disqualifying on their own."
+      }
+    },
+    "technicalDetails": "Risk Assessment Details",
+    "technicalDetailsDescription": "Potential risks this third-party app may introduce to your organization, by risk category.",
+    "appStore": "App Store",
```

Biome (2.5.3)

[error] The key appStore was already declared.

(lint/suspicious/noDuplicateObjectKeys)